.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,143 @@
+      placeholder: "14.8"
+    validations:
+      required: true
+  - type: input
+    id: terraform-version
+    attributes:
+      label: Terraform version
+      description: What version of terraform are you using?
+      placeholder: "Please paste the output of `terraform -v`` here"
+    validations:
+      required: true
+  - type: dropdown
+    id: affected-resources
+    attributes:
+      label: Affected Resources / Data Sources
+      description: What resources and / or data sources are affected?
+      multiple: true
+      options:
+        - other
+        - datasource/gitlab_group_membership
+        - datasource/gitlab_project
+        - datasource/gitlab_project_protected_branch
+        - datasource/gitlab_project_protected_branches
+        - datasource/gitlab_projects
+        - datasource/gitlab_user
+        - datasource/gitlab_users
+        - resource/gitlab_branch_protection
+        - resource/gitlab_deploy_key
+        - resource/gitlab_deploy_key_enable
+        - resource/gitlab_deploy_token
+        - resource/gitlab_group
+        - resource/gitlab_group_access_token
+        - resource/gitlab_group_badge
+        - resource/gitlab_group_cluster
+        - resource/gitlab_group_custom_attribute
+        - resource/gitlab_group_label
+        - resource/gitlab_group_ldap_link
+        - resource/gitlab_group_membership
+        - resource/gitlab_group_share_group
+        - resource/gitlab_group_variable
+        - resource/gitlab_instance_cluster
+        - resource/gitlab_instance_variable
+        - resource/gitlab_label
+        - resource/gitlab_managed_license
+        - resource/gitlab_pipeline_schedule
+        - resource/gitlab_pipeline_schedule_variable
+        - resource/gitlab_pipeline_trigger
+        - resource/gitlab_project
+        - resource/gitlab_project_access_token
+        - resource/gitlab_project_approval_rule
+        - resource/gitlab_project_badge
+        - resource/gitlab_project_cluster
+        - resource/gitlab_project_custom_attribute
+        - resource/gitlab_project_freeze_period
+        - resource/gitlab_project_hook
+        - resource/gitlab_project_level_mr_approvals
+        - resource/gitlab_project_membership
+        - resource/gitlab_project_mirror
+        - resource/gitlab_project_share_group
+        - resource/gitlab_project_variable
+        - resource/gitlab_repository_file
+        - resource/gitlab_service_github
+        - resource/gitlab_service_jira
+        - resource/gitlab_service_microsoft_teams
+        - resource/gitlab_service_pipelines_email
+        - resource/gitlab_service_slack
+        - resource/gitlab_tag_protection
+        - resource/gitlab_user
+        - resource/gitlab_user_custom_attribute
+    validations:
+      required: false
+  - type: textarea
+    id: configuration
+    attributes:
+      label: Relevant Terraform Configuration
+      description: Please copy and paste the relevant Terraform Configuration here
```

Comment:
```suggestion
      description: Please copy and paste the relevant Terraform Configuration here. This will be automatically formatted into code, so no need for backticks.
```